src/smc-hub/hub.coffee

```diff
@@ -506,6 +498,14 @@ exports.start_server = start_server = (cb) ->
         (cb) ->
             # This must happen *AFTER* update_schema above.
             init_smc_version(database, cb)
+        (cb) ->
```

Comment:
Why?  This change has nothing to do with kiosk mode, etc., so why did you make it? Is it fixing a bug that you just happened to notice while doing something else?